website/docs/api/bigint-money/createBigIntMoney/allocateTo.md

```diff
@@ -5,7 +5,16 @@ hide_title: true
 sidebar_label: allocateTo
 ---
 
-# allocateTo
+# `allocateTo(number)`
+
+#### Arguments
+
+1. `number`
```

Comment:
you should describe the type of input param, for example, if the type is MoneyBase we add MoneyBase if string we pass a string and so on

For example
http://localhost:3000/docs/api/bigint-money/createBigIntMoney/allocateTo
must be a
number (number)

If we have a string we pass string and so on

Reference
https://effector.now.sh/docs/api/effector/createEffect

or like here
http://localhost:3000/docs/api/money/createMoney/divide